.github/workflows/ci_test-full.yml

```diff
@@ -31,11 +31,6 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
-    - name: Update Pip
-      run: |
-        # todo: unfreeze PIP after resolving minimal dependencies
-        pip install --quiet "pip==20.1" --upgrade --user  # needed for get pip cacher folder
```

Comment:
we shall keep it, just allow newer versions `pip>=20.1` as it is needed for the proper function of caching step